stages/node-fast.yaml

```diff
@@ -84,10 +84,10 @@ spec:
       nodeInfo:
         architecture: {{ with .architecture }} {{ . }} {{ else }} "amd64" {{ end }}
         bootID: {{ with .bootID }} {{ . }} {{ else }} "" {{ end }}
-        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "" {{ end }}
-        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "" {{ end }}
-        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "fake" {{ end }}
-        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "fake" {{ end }}
+        containerRuntimeVersion: {{ with .containerRuntimeVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kernelVersion: {{ with .kernelVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kubeProxyVersion: {{ with .kubeProxyVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
+        kubeletVersion: {{ with .kubeletVersion }} {{ . }} {{ else }} "kwok{{ Version }}" {{ end }}
```

Comment:
This also requires the implementation of `Version` template function

Reply:
Where I can define ther template function of `Version`？Can you give me an example of something like this.